content/docs/state-and-lifecycle.md

```diff
@@ -409,41 +401,41 @@ Then you can update them independently with separate `setState()` calls:
   }
 ```
 
-The merging is shallow, so `this.setState({comments})` leaves `this.state.posts` intact, but completely replaces `this.state.comments`.
+Միավորումը մակերեսային է, այսինքն\` `this.setState({comments})`-ը թողնում է `this.state.posts`-ին «անվնաս», բայց ամբողջությամբ փոխարինում է `this.state.comments`-ը։
 
-## The Data Flows Down {#the-data-flows-down}
+## Տվյալների հոսքը ներքև {#the-data-flows-down}
 
-Neither parent nor child components can know if a certain component is stateful or stateless, and they shouldn't care whether it is defined as a function or a class.
+Ո'չ ծնող, ո'չ զավակ կոմպոնենտները չեն կարող իմանալ, արդյոք տվյալ կոմպոնենտը վիճակով(stateful) է կամ վիճակազուրկ(stateless), և նրանք չպետք է հոգ տանեն այն բանի մասին, թե դա հայտարարված է որպես ֆունկցիա թե որպես կլաս։
 
-This is why state is often called local or encapsulated. It is not accessible to any component other than the one that owns and sets it.
+Ահա թե ինչու state-ը հաճախ անվանվում է լոկալ կամ ինկապսուլացված։ Այն հասանելի չէ ուրիշ ոչ մի կոմպոնենտից, բացի այն մեկից, որում հենց ինքն է և որը տեղադրել է իրեն։
```

Comment:
"the one that owns and sets it." ստեղ owns it-ը ես հասկանում եմ "որին պատկանում է", այսինքն որը, որ իրա տերն ա, ոչ թե "որում հենց ինքն է"